src/armlist.js

```diff
@@ -348,6 +349,12 @@ var Arm = CreateClass({
         this.props.onChange(this.props.id, state, false);
     },
     treatAddArmFromTemplate: function (state, newarm, considerNum) {
+        var pattern = new RegExp("(\\d+)","i");
+        var matches = pattern.exec(newarm.imageURL);
```

Comment:
`var matches = newarm.imageURL.match(/(\d+)/);`

no need ignore case option for number.